src/map.h

```diff
@@ -492,6 +492,28 @@ class map
         bool clear_path( const tripoint &f, const tripoint &t, const int range,
                          const int cost_min, const int cost_max ) const;
 
+
+        /**
+         * Populates a vector of points that are reachable within a number of steps from a
+         * point. It could be generalized to take advantage of z levels, but would need some
+         * additional code to detect whether a valid transition was on a tile.
+         *
+         * Does the following:
+         * 1. Checks if a point is reachable using a flood fill and if it is, adds it to a vector.
+         *
+         */
+        void map::reachable_flood_steps( std::vector<tripoint> &reachable_pts, const tripoint &f,
```

Comment:
function part of `map`, `map::` qualification is throwing compile errors
```suggestion
        void reachable_flood_steps( std::vector<tripoint> &reachable_pts, const tripoint &f,
```